preprocessors/openstreetmap.schema.json

```diff
@@ -0,0 +1,138 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/openstreetmap.schema.json",
+    "type": "object",
+    "title": "OpenStreetMap Preprocessor",
+    "description": "Information on the contents of a place on a map",
+    "properties": {
+
+        "timestamp": {
+            "type": "number",
+            "minimum": 0
+        },
+        "request_uuid": {
+            "type": "string",
+            "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-4[0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+        },
+        "name": {
+            "type": "string",
+            "pattern": "^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-)))+$"
+        },
+        "data": {
+            "type": "object",
+            "properties": {
+                "point_of_interest": {
```

Comment:
Is the field really `point_of_interest` singular although it represents a list of points? This seems like a point of confusion to me. I'd recommend it be renamed `points_of_interest` unless there's a reason otherwise (e.g., the name comes from a similar structure in OSM).